solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -385,6 +388,9 @@ public CoreContainer(NodeConfig config, CoresLocator locator, boolean asyncSolrC
       }
     }
 
+    this.allowListUrlChecker = AllowListUrlChecker.create(config);
+    log.info("URL allow-list initialized: {}", this.allowListUrlChecker);
```

Comment:
I question that this worth logging at INFO for.  Solr has *tons* of plugins.

Reply:
I agree. I copied it from the old WhitelistChecker, but that's probably too much.